README.md

```diff
@@ -21,12 +21,13 @@ development build and reporting any issues you run into.
 > Prebuilt images for cert-manager are made available on Dockerhub.
```

Comment:
❓ This gets rendered as quoted text. Is that intended?

Reply:
Yep